### PR TITLE
Qr

### PR DESCRIPTION
**Pull request by MohiTheFish**
Creates endpoint /qr where users' qr codes show up. 

Also turned off eslint's linebreak-style cause I didn't think it mattered.

### CODE REVIEW

File: src/components/QR/qr.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+#qrimg {
+    margin: 0;
+    padding: 0;
+    width: 200px;
+    height: 200px;
+}
+
+@media screen and (min-width: 320px) {
```
Comment (brianstrauch) [inline src/components/QR/qr.scss]: Let's just make a desktop version and a mobile version. No need to handle a bunch of different sized devices

File: src/components/QR/index.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/*eslint-disable*/
```
Comment (brianstrauch) [inline src/components/QR/index.jsx]: Does this disable all linting rules?

**Review by brianstrauch — COMMENTED** [review]
Looks good! I can't see the actual QR because of a CORS error :P, but you'll have to show me what it looks like in person

File: src/components/QR/index.jsx (hunk 2 above)
Comment (MohiTheFish) [reply src/components/QR/index.jsx]: It disables it for the page. Completely forgot I left that there ooooops.

**Review by brianstrauch — APPROVED** [review]

**Review by abhinavpappu — APPROVED** [review]

**Pull request merged**